site/layouts/partials/home/masthead-followup.html

```diff
@@ -34,7 +34,9 @@ <h2 class="display-5 font-weight-normal">BootstrapCDN</h2>
     <div class="col-md-7 pl-md-5">
       {{ highlight (printf (`<!-- CSS only -->
 <link rel="stylesheet" href="%s" integrity=%q crossorigin="anonymous">`) .Site.Params.cdn.css (.Site.Params.cdn.css_hash | safeHTMLAttr)) "html" "" }}
-      {{ highlight (printf (`<!-- JavaScript and dependencies -->
+      {{ highlight (printf (`<!-- JavaScript Bundle with Popper.js -->
+<script src="%s" integrity=%q crossorigin="anonymous"></script>`) .Site.Params.cdn.js_bundle (.Site.Params.cdn.js_bundle_hash | safeHTMLAttr)) "html" "" }}
+      {{ highlight (printf (`<!-- Or Popper.js and our JavaScript file separately -->
```

Comment:
I kept this as a new block so that it can be copied separately

Reply:
Dropping this from the homepage. We only need to show one option on the homepage.